app/rww/ldp/WebClient.scala

```diff
@@ -236,9 +235,7 @@ class WSClient[Rdf <: RDF](readerSelector: ReaderSelector[Rdf, Try], rdfWriter:
     }
   }
 
-  override
-  def put[S](url: Rdf#URI, graph: Rdf#Graph, syntax: Syntax[S])
-            (implicit writer: Writer[Rdf#Graph, Try, S]): Future[Unit] = {
+  override def put[S](url: Rdf#URI, graph: Rdf#Graph, syntax: Syntax[S])(implicit writer: Writer[Rdf#Graph, Try, S]): Future[Unit] = {
```

Comment:
yeah, we have not been too many on the project at present. I should try to run a generalised formatter on the whole thing, to have a base to work from.